test/csuite/random_abort/main.c

```diff
@@ -217,6 +218,17 @@ thread_run(void *arg)
         if (fprintf(fp[INSERT_RECORD_FILE_ID], "%" PRIu64 "\n", i) == -1)
             testutil_die(errno, "fprintf");
 
+        /*
+         * If configured, run compaction on database after each epoch of 100000 operations.
+         */
+        if (i >= 100000 && i % 100000 == 0 && compaction) {
```

Comment:
Minor suggestion: Test `compaction` first in the conditional. No need to perform expensive modulo operations on every iteration for every thread if compaction is turned off.

Reply:
Fixed.